service/src/LevelZero.cpp

```diff
@@ -16,6 +16,11 @@
 
 #include "LevelZeroImp.hpp"
 
+static void __attribute__((constructor)) geopm_levelzero_init(void)
+{
+    setenv("ZES_ENABLE_SYSMAN", "1", 1);
+}
+
```

Comment:
This was just moved into geopm.service.  Did you intend to bring this back?  Can we not set this in the environment of the calling process rather than calling setenv?

Reply:
Accidental addition. Thanks for catching it. Removed.